app/src/test/kotlin/compiler/syntax/AstFactoryTest.kt

```diff
@@ -228,6 +281,28 @@ class AstFactoryTest {
                 makeTNode(TokenType.IDENTIFIER, "poboczna3")
             ),
             makeTNode(TokenType.NEWLINE, "\n"),
+            makeNTNode(
+                NonTerminalType.FOREIGN_DECL, Productions.foreignDecl,
+                makeTNode(TokenType.FOREIGN, "zewnętrzna"),
```

Comment:
"Zewnętrzna przekaźnik" sounds odd. Guess we wouldn't have many foreign generators, so it's not a big issue, but I think it is a place for improvements, possibly in next sprints.

Reply:
What do you think about changing to `zew`?

Reply:
For now, I changed to `zew`, let me know what do you think.

Reply:
We can just add another keyword `zewnętrzny`.

Reply:
For me we `zew` is good enough, or we can add `zewnętrzny` with grammar rules so that it matches only `przekaźnik` (and `zewnętrzna` only `czynność`).

Reply:
If I was a user of this language then I would really prefer not to be able to make such “hard to detect” errors. So I would go with “zew” unless someone strongly opposes.

Reply:
I would just add `zewnętrzny` as a synonym for `zewnętrzna` (both usable before `czynność` and `przekaźnik`) so that the user could write it nicely, but also would not have to bother with those "hard to detect" errors. If you really want a single keyword for it, then I would suggest to change it to more suggestive `zewn`.

Reply:
This sounds like a nice solution, I will add two options.

Reply:
(Jakże ważna dyskusja się tu zrobiła xd)